consistency/tool/comparison/comparison.py

```diff
@@ -0,0 +1,32 @@
+class Comparison:
```

Comment:
I am fine with merging this and iterating over this structure as we develop the comparison part of the tool.